tests/Query/ExpressionTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ClickHouseDB\Tests\Query;
+
+use ClickHouseDB\Query\Expression;
+use ClickHouseDB\Quote\FormatLine;
+use PHPUnit\Framework\TestCase;
+
+class ExpressionTest extends TestCase
```

Comment:
```suggestion
final class ExpressionTest extends TestCase
```

Reply:
Is it essential? I mean it doesn't add any value :) And changes nothing.

Reply:
I several codebases I met accidentaly extended Test classes instead of TestCase classes (result of huge refactorings, renaming etc.). This simply prevents that so you're safe, your tests won't ever be extended ;)

Reply:
I see. Now it makes sense.